templates/partials/nav.html

```diff
@@ -0,0 +1,19 @@
+<nav class="navigation">
+  <div>
+    <a href="/"><img class="navigation__logo" src="/dist/images/google-go.jpg"></img></a>
```

Comment:
This is another example of when a background image should be used instead:

```html
<div class="navigation__logo">
  <a href="/"><span class="sr-only>">Go Google Scraper</span></a>
</div>
```

> `sr-only` is for screen-reader so for accessibility

Reply:
I will update the usage of images and svgs in this [issue](https://github.com/gutakk/go-google-scraper/issues/41)